pom.xml

```diff
@@ -137,6 +137,9 @@
         <jackson.databind.version>2.10.0</jackson.databind.version>
         <kafka.version>2.4.0</kafka.version>
         <janus.version>0.4.0</janus.version>
+        <snappy.version>1.1.7.2</snappy.version>
+        <antlr.version>3.5.2</antlr.version>
```

Comment:
Antlr 3 dates back to 2014 and is likely to have security issues. Please update to antlr 4 ie 4.8-1 or document why antlr3 is required

Reply:
This version is required by janusgraph. Can test with a more recent one, but even if it is successful, I do not see it as a full proof solution

Reply:
thanks - looking at options here. will update

Reply:
antlr3 is being used by a variety of components - janus itself, and then it's other packages ie for cassandra, tinkerpop, elastic search
I concur, let's go with that version. I'll take a look at the dependency reports for CVEs and if we need to do any further changes we'll do in a new PR

Reply:
snappy occurs in a few more places, including kafka. However we are close to current there, so less of a concern. Again, agree, let's go with this and can reevaluate the cve/dependency report after